xbmc/Application.cpp

```diff
@@ -297,7 +297,17 @@ void CApplication::HandlePortEvents()
         break;
       case XBMC_VIDEOMOVE:
       {
-        CServiceBroker::GetWinSystem()->OnMove(newEvent.move.x, newEvent.move.y);
+        const auto winSystem = CServiceBroker::GetWinSystem();
+        winSystem->OnMove(newEvent.move.x, newEvent.move.y);
+
+        // Not save position if is fullscreen to not overwrite last valid windowed position
+        if (!winSystem->IsFullScreen())
+        {
+          const auto settings = CServiceBroker::GetSettingsComponent()->GetSettings();
+          settings->SetInt(CSettings::SETTING_WINDOW_TOP, newEvent.move.y);
+          settings->SetInt(CSettings::SETTING_WINDOW_LEFT, newEvent.move.x);
+          settings->Save();
```

Comment:
i'm no sure but this may cause frequently writes to settings file with no reason.

Reply:
This event only is generated one time when window has finished to move (not continually while user is moving window):

https://github.com/xbmc/xbmc/blob/5dda244747b86975de9ac4e104a4e981c535a509/xbmc/windowing/windows/WinEventsWin32.cpp#L575

Also is generated one time when window switch from fullscreen to windowed (or windowed to fullscreen but is not  taken into account in this case). Then no frequent writes can occur.